### PR TITLE
long running job support

### PR DESCRIPTION
**Pull request by zfengms**

### CODE REVIEW

File: R/cluster.R [hunk 1]
```diff
@@ -203,27 +212,60 @@ makeCluster <-
...
         ),
         fill = TRUE
       )
 
-
       clusterNodeMismatchWarning <-
-        paste0("There is a mismatched between the projected cluster %s ",
-               "nodes min/max '%s'/'%s' and the existing cluster %s nodes '%s'")
+        paste(
+          "There is a mismatched between the projected cluster %s",
```
Comment (paselem) [inline R/cluster.R]: change 'projected' to 'requested'

File: R/doAzureParallel.R [hunk 2]
```diff
@@ -181,14 +195,24 @@ setVerbose <- function(value = FALSE) {
...
-                 "job",
-                 time)
-
   if (!is.null(obj$options$azure$job)) {
     id <- obj$options$azure$job
   }
+  else {
+    time <- format(Sys.time(), "%Y%m%d%H%M%S", tz = "GMT")
+    id <-  sprintf("%s%s",
+                   "job",
```
Comment (paselem) [inline R/doAzureParallel.R]: Lines 202 and 203 have very different styles. One has all the params on one line, the other has it split (for roughly the same number of characters on a single line). Why?

File: R/doAzureParallel.R [hunk 3]
```diff
@@ -307,13 +331,50 @@ setVerbose <- function(value = FALSE) {
...
+      }
+
+      if (!is.null(obj$options$azure$chunksize)) {
+        chunkSize <- obj$options$azure$chunksize
+      }
+
+      if (exists("chunkSize", envir = .doAzureBatchGlobals)) {
+        chunkSize <- get("chunkSize", envir = .doAzureBatchGlobals)
+      }
+
```
Comment (paselem) [inline R/doAzureParallel.R]: nitpick: remove extra line

File: R/doAzureParallel.R (hunk 2 above)
Comment (zfengms) [reply R/doAzureParallel.R]: Not sure why, I updated line 203 to 1 line, and did a reformat again and it was kept in one line, the inconsistency might be due to manual update.

File: R/doAzureParallel.R (hunk 3 above)
Comment (zfengms) [reply R/doAzureParallel.R]: removed.

File: R/cluster.R (hunk 1 above)
Comment (zfengms) [reply R/cluster.R]: fixed.

**Pull request merged**